Formula/ser2net.rb

```diff
@@ -21,35 +21,39 @@ class Ser2net < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "2c4726875c7f661b925b6fbd7d82eeaf89fd4b91a28ba34377860c8ebbb3f999"
   end
 
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
   depends_on "libyaml"
 
   resource "gensio" do
-    url "https://downloads.sourceforge.net/project/ser2net/ser2net/gensio-2.1.4.tar.gz"
-    sha256 "1f5a29aabfb35886893cfda5cd78192db67e96de796dbf9758dbecd4077a3fd8"
-
-    # Fix -flat_namespace being used on Big Sur and later.
-    patch do
-      url "https://raw.githubusercontent.com/Homebrew/formula-patches/03cf8088210822aa2c1ab544ed58ea04c897d9c4/libtool/configure-big_sur.diff"
-      sha256 "35acd6aebc19843f1a2b3a63e880baceb0f5278ab1ace661e57a502d9d78c93c"
-    end
+    url "https://github.com/cminyard/gensio.git"
```

Comment:
We should at least fix a revision here, if not a tag.

Reply:
I will; a new release is due soon. This PR is in draft mode for the benefit of [this discussion](https://sourceforge.net/p/ser2net/discussion/90083/thread/f3ae30894e/).